rendezvous/README.md

```diff
@@ -234,3 +249,17 @@ message Message {
   optional DiscoverResponse discoverResponse = 6;
 }
 ```
+
+## Recommendations for Rendezvous Points configurations
+
+Rendezvous points should have well defined configurations to enable libp2p nodes running the rendezvous protocol to have friendly defaults, as well as to guarantee the security and efficiency of a Rendezvous point. This will be particularly important in a federation, where rendezvous points should share the same expectations.
```

Comment:
Nit pick: The rest of the specification is wrapped at 80 characters. Can you do the same here for consistency?

Reply:
fixed